03/app.js

```diff
@@ -42,10 +43,32 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
+    
+    const title = this[isbn]['title'];
+
+    if (title[lang]) {
```

Comment:
Najpierw trzeba sprawdzić czy `this[isbn]`  istnieje, następnie `this[isbn]['title']` a potem `lang` dopiero :P